third-party/pybind11/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#  minimum required cmake version: 3.1.0
+cmake_minimum_required(VERSION 3.1.0)
+
+project(pybind11-download NONE)
+
+include(ExternalProject)
+ExternalProject_Add(
+        pybind11
+        PREFIX third-party/pybind11
+        GIT_REPOSITORY "https://github.com/pybind/pybind11.git"
+        GIT_TAG "f1abf5d9159b805674197f6bc443592e631c9130" # v2.6.1
+        SOURCE_DIR "${CMAKE_BINARY_DIR}/third-party/pybind11/src"
```

Comment:
I'm still unclear about this...
If we made this `${CMAKE_BINARY_DIR}/third-party/pybind11`, what would happen?

Reply:
I tried all kind of deployments,
If I do what you asked I get lots of cmake errors.

**Examples:**
 Error copying file "C:/Work/Git/librealsense/build/third-party/pybind11/configure/third-party/pybind11/src/pybind11-stamp/pybind11-gitinfo.txt" to "C:/Work/Git/librealsense/build/third-party/pybind11/configure/third-party/pybind11/src/pybind11-stamp/pybi
nd11-gitclone-lastrun.txt".


  CMake Error at pybind11/tmp/pybind11-gitclone.cmake:64 (message):

    Failed to copy script-last-run stamp file:

    'C:/Work/Git/librealsense/build/third-party/pybind11/configure/third-party/pybind11/src/pybind11-stamp/pybind11-gitclone-lastrun.txt'